### DISCUSSION TITLE
document_exporter command not found

### DISCUSSION THREAD

**Discussion by Stevelande** [tteck/Proxmox#1683] [Help Needed]
Good day. I have paperless ngx via the script 
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx.sh)" installed. It also runs great. But without backup strategy I don't want to use it fully. But I can not use the document_exporter. Command not found. Do not understand the installation correctly either. Is Peperless installed as a docker image? Or how is it installed?
Thanks

Translated with www.DeepL.com/Translator (free version)

**Comment by tteck** [comment]
> But without backup strategy I don't want to use it fully.

Scroll down to "Proxmox Backups"
https://smarthomescene.com/guides/how-to-backup-home-assistant-in-proxmox/

> Is Peperless installed as a docker image? Or how is it installed?

The installation is performed as bare-metal, not through Docker.

**Comment by tabasavr** [comment]
I got export to work by doing this in LXC console:
```
cd /opt/paperless/src
/usr/bin/python3 manage.py document_exporter [your args here]
```
